### DISCUSSION TITLE
Default-Values Date, Time now in FS

### DISCUSSION THREAD

**Discussion by teamtacho** [liedekef/events-made-easy#623] [General]
Hi Team
how can I set a default value for the start-time and start-date as current time and date?

**Comment by liedekef** [comment]
The next version of EME will have a new parameter for the shortcode that allows you to set this. See https://github.com/liedekef/events-made-easy/commit/55aef790d1fd230842a34496961b6b9389c62cef (only the changes to eme-fs.php are relevant)
